packages/authentication/ios/Plugin/FirebaseAuthentication.swift

```diff
@@ -645,6 +646,16 @@ public typealias AuthStateChangedObserver = () -> Void
         return self.config
     }
 
+    func requestAppTrackingTransparencyPermission(completion: @escaping () -> Void) throws {
```

Comment:
```suggestion
    func requestAppTrackingTransparencyPermission(completion: @escaping () -> Void) {
```

`throws` is not needed here, right?

Reply:
It was not needed. I removed it.